packages/dds/tree/src/feature-libraries/modular-schema/crossFieldQueries.ts

```diff
@@ -82,4 +83,21 @@ export interface CrossFieldManager<T = unknown> {
 		newValue: T,
 		invalidateDependents: boolean,
 	): void;
+
+	/**
+	 * This must be called whenever a new node is moved into this field as part of the current rebase, compose, or invert.
+	 * Calling this for a node which was already in the field is tolerated.
+	 */
+	moveNode(id: NodeId): void;
+
+	/**
+	 * This must be called whenever a new cross field key is moved into this field as part of the current rebase or compose.
```

Comment:
Is a cross field key defined by the combination of a CrossFieldTarget, RevisionTag, and ChangesetLocalId? I wonder if would be nicer to make a type for it. Up to you.

Reply:
Ah, I see you did that elsewhere already.